Exit with a special value when RNG modules are not available

Instead of returning with an error, exit with a special value when the
RNG kernel modules are not present in the kernel. This is not really a
hard failure.

Patch borrowed from Fedora, at
http://pkgs.fedoraproject.org/cgit/rng-tools.git/tree/rngd-exit-code.patch.

Signed-off-by: Maxim Mikityanskiy <maxtram95@gmail.com>

--- a/rngd.c
+++ b/rngd.c
@@ -319,7 +319,7 @@ int main(int argc, char **argv)
 			message(LOG_DAEMON|LOG_ERR,
 				"Maybe RNG device modules are not loaded\n");
 		}
-		return 1;
+		return 66;
 	}
 
 	if (arguments->verbose) {
